src/map/latent_effect.h

```diff
@@ -36,6 +36,7 @@ enum LATENT
     LATENT_HP_OVER_VISIBLE_GEAR     = 46, //hp more than or equal to %, calculated using HP bonuses from visible gear only 
     LATENT_MP_UNDER_PERCENT         = 4,  //mp less than or equal to % - PARAM: MP PERCENT
     LATENT_MP_UNDER                 = 5,  //mp less than # - PARAM: MP #
+    LATENT_MP_OVER                  = 49, //mp greater than # - PARAM: MP #

```

Comment:
omfg..
@Shadowmh please forgive me. It shroud have stayed 55.

On the original diff I was looking it, the list truncated and when I hit the expand view on github it didn't expand as far as I thought it did. I'm very sorry. Do you know how to collapse commits or how to roll back history? if not I can show you realty quick.

`#define MAX_LATENTEFFECTID 55` means 54 was the last ID in use, so next one is 55 and the max has to increment to 56.